routes/user.js

```diff
@@ -30,8 +30,12 @@ router.post("/add", urlEncodedParser, async (req, res) => {
   console.log(url); // Temporary for debugging.
 
   try {
-    await EmailUtils.sendEmail(email, "Firefox Breach Alert",
-      `Visit this link to subscribe: ${url}`);
+    await EmailUtils.sendEmail(
+      email,
+      "Verify your email address to subscribe to Firefox Monitor.",
+      "email_verify",
+      { email: email, url: url}
```

Comment:
Not sure if we want some ESLint rule for this, but we can use shorthand notation here:

```js
{ email, url }
```

Ref: https://eslint.org/docs/rules/object-shorthand

Reply:
Hmm ... I think I like `object-shorthand` when all of the object properties' values have the same name, but not when there's a mix of same-name values and others. So I'll fix this one, but won't add the rule to `.eslintrc`.